eventmesh-common/src/main/java/org/apache/eventmesh/common/MetricsConstants.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.common;
+
+
+public abstract class MetricsConstants {
+
+    private MetricsConstants() {
+
+    }
```

Comment:
Maybe decorating this constant with `class` is enough, instead of `abstract class`.

The usages of this constant are all in metrics-plugin module, so perhaps placing this class in metrics-plugin module will be better.

>This comment is a copy of https://github.com/apache/eventmesh/pull/3446#discussion_r1450991643

Reply:
Fixed in https://github.com/apache/eventmesh/pull/4840/files/8936c98ed32c8c7a43c1448a77320f102f90ff25..1508e4c24c9d2d78b696b50e93f5bbf3b7dd1446